postgresql.sql

```diff
@@ -0,0 +1,15 @@
+--http://sqlfiddle.com/#!15/9eecb7db59d16c80417c72d1e1f4fbf1/11608
+--Runs with postgresql 9.3
+--This will not match with the expected output because of the column header called "FizzBuzzOrBoth" !
```

Comment:
There might be, but I felt that its not so important, I mean it's just a header and aren't we all used to using the language in that way - always have headers in the output of the queries? :)